homeassistant/helpers/selector.py

```diff
@@ -87,6 +152,10 @@ def serialize(self) -> dict[str, dict[str, _T]]:
         vol.Optional("domain"): vol.All(cv.ensure_list, [str]),
         # Device class of the entity
         vol.Optional("device_class"): vol.All(cv.ensure_list, [str]),
+        # Features supported by the entity
+        vol.Optional("supported_features"): [
+            vol.All(vol.Any(int, str), _validate_supported_feature)
```

Comment:
let's not support int. It's difficult to read

Reply:
This is required to be able to instantiate a selector from an already validated configuration where the enum strings have been resolved to their integer values.
We do that in tests to test serialization of configuration, I'm not sure if it's actually needed though?